test/e2e/framework/fixture.go

```diff
@@ -609,20 +609,17 @@ func (sf *StartedSyncerFixture) WaitForClusterReadyReason(t *testing.T, ctx cont
 	kcpClusterClient, err := kcpclient.NewClusterForConfig(cfg.UpstreamConfig)
 	require.NoError(t, err)
 	kcpClient := kcpClusterClient.Cluster(cfg.KCPClusterName)
-	require.Eventually(t, func() bool {
+	Eventually(t, func() (bool, string) {
```

Comment:
not related to here: can we document `Eventually` and the pitfalls it solves vs. using `require.Eventually` directly?

Reply:
done as part of other PR.